aca_preview/preview.py

```diff
@@ -30,7 +35,26 @@
         setattr(CHAR, attr, getattr(GUIDE, attr))
 
 
-def preview_load(load_name, outdir=None):
+def main(sys_args=None):
+    """Command line interface to preview_load()"""
+
+    import argparse
+    parser = argparse.ArgumentParser(description='ACA preliminary review tool')
+    parser.add_argument('load_name',
+                        type=str,
+                        help='Load name (e.g. JAN2119A)')
```

Comment:
This looks good.  Me, I prefer filename for better natural tab completion and being able to call on a pickle conveniently outside my working directory, but 'load_name' may continue to make the most sense for product directories.

Reply:
Fair enough.  At risk of over-complicating things, I added 1bb7278 for your tab-completion pleasure.